packages/emails/src/components/Info.tsx

```diff
@@ -7,7 +7,7 @@ export const Info = (props: {
   withSpacer?: boolean;
   lineThrough?: boolean;
 }) => {
-  if (!props.description) return null;
+  if (!props.description || props.description=="") return null;
```

Comment:
Some cases like additionlNotes are set to "" and was causing a random header to appear in the email field